pkg/ebpf/gadgets/http/tracer/tracer.go

```diff
@@ -103,7 +103,7 @@ func (t *Tracer) install() error {
 	for _, tp := range tracepoints {
 		l, err := tracepointlib.AttachTracepoint(tp)
 		if err != nil {
-			logger.L().Error(fmt.Sprintf("Error attaching tracepoint: %s", err))
+			logger.L().Error("http Tracer - error attaching tracepoint", helpers.Error(err)) // TODO Fatal? check with Amit
```

Comment:
It can be Fatal, if we do Fatal on the rest of the tracers we can also do it here.

Reply:
we do errors when attaching in ptrace - but in practice do we really want to run node-agent if we cannot attach here?

Reply:
Probably not, let's make it Fatal.

Reply:
ok, I put fatal for both